### PR TITLE
[Term Entry] array_reduce() PHP

### LINKED ISSUE #3045: [Term Entry] PHP Arrays `array_reduce()`
### Reason/inspiration (optional)

We would like a new term entry for [PHP](https://www.codecademy.com/resources/docs/php) about the `array_reduce()` function.

The entry should go in a new file in `docs/content/php/concepts/arrays/terms/array-reduce/array-reduce.md` (Note the dashes rather than underscores.)

The entry should 

- Have a syntax section illustrating the function with a ` ```pseudo ` code block. 
- An example showing the function in use with a ` ```php ` code block. 
- A runnable codebyte example in a ` ```codebyte/php` block. 

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by jaw163**
### Description

Term entry for PHP's array_reduce function.

Closes #3045 

### Type of Change

Updating documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by SSwiniarski** [comment]
@jaw163, you never asked to be assigned this issue. We require contributors to ask to be assigned an issue and be officially assigned before submitting a PR. Fortunately, this time there wasn't another person working on this issue so I can assign you now. But please be aware that you should be assigned first.

File: content/php/concepts/arrays/terms/array-reduce/array-reduce.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    }
+
+    var_dump(array_reduce($arrOfNumbers, "highestNumber")); // int(23)
+
+?>
+```
+
+## Codebyte Example
+
+Here is an example for you to try:
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-reduce/array-reduce.md:55]:
```suggestion
The following code is runnable and demonstrates the use of `array_reduce()`:
```

File: content/php/concepts/arrays/terms/array-reduce/array-reduce.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+```codebyte/php
+<?php
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-reduce/array-reduce.md:58]: I would suggest replacing this with another example, I don't think this function does a good job of highlighting a common use case of this method

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@jaw163 I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

File: content/php/concepts/arrays/terms/array-reduce/array-reduce.md (hunk 2 above)
Comment (jaw163) [reply content/php/concepts/arrays/terms/array-reduce/array-reduce.md:63]: I just wanted to show an example where an initial value was used. I've included a different one that does something similar though

**Review by caupolicandiaz — APPROVED** [review]
@jaw163 Overall, looks good. I made some direct edits to adjust some spacing and readability issues. This is ready for a second review.

**Pull request merged**